Source/WebKit/UIProcess/Extensions/Cocoa/API/WebExtensionContextAPIActionCocoa.mm

```diff
@@ -166,6 +166,12 @@
         return;
     }
 
+    RefPtr controller = extensionController();
+    if (controller && controller->isShowingActionPopup()) {
```

Comment:
`extensionController()` should always be non-null here, since the extension is loaded and there is earlier checks outside here that test this.

```
if (extensionController()->isShowingActionPopup()) {
```